proxy.go

```diff
@@ -99,6 +98,11 @@ func (rp *reverseProxy) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		ReadCloser: req.Body,
 	}
 
+	// publish session_id if needed
+	if s.sessionId != "" {
+		rw.Header().Set("X-ClickHouse-Server-Session-Id", s.sessionId)
```

Comment:
Quick check didn't show any results for "X-ClickHouse-Server-Session-Id" header in CH docs. Could you pls add a comment what exactly this header is used for?

Reply:
I added this header just to make sure ChProxy acknowledged and processed correctly the value was set to session_id

Reply:
I was confused while reading it. I think, it requires a proper comment to explain your intention.

Reply:
@hagen1778 treat this header as ECHO Service, you pass the value of session_id and you'll want to assure that that value was accepted by server-side correctly.

Reply:
@hagen1778 let's merge it already and let's enable GitHub actions to release versions properly for multiple platforms, what do u think?